packages/components/src/DescriptionList/DescriptionList.test.tsx

```diff
@@ -14,6 +14,21 @@ it("renders an object as a list of key value pairs", () => {
   expect(container).toMatchSnapshot();
 });
 
+it("allows duplicate terms", () => {
+  const logSpy = jest.spyOn(global.console, "error");
+
+  const { container } = render(
+    <DescriptionList
+      data={[
+        ["Issued", "2018-12-08"],
+        ["Issued", "2019-01-06"],
+      ]}
+    />,
+  );
+  expect(container).toMatchSnapshot();
```

Comment:
while you're updating this, i'd like to not add more snapshots so if we can 

```ts
expect(getAllByText("Issued")).toHaveLength(2)
```

that would be great!